src/api/app/models/review.rb

```diff
@@ -101,23 +105,23 @@ def check_initial
       errors.add(:base, 'it is not allowed to have more than one reviewer entity: by_user, by_group, by_project, by_package.')
     end
 
-    if by_user && !reviewable
+    if by_user && !user
       errors.add(:by_user, "#{by_user} not found")
     end
 
-    if by_group && !reviewable
+    if by_group && !group
       errors.add(:by_group, "#{by_group} not found")
     end
 
-    if by_project && !reviewable
+    if by_project && !project
       # must be a local project or we can't ask
       errors.add(:by_project, "#{by_project} not found")
     end
 
-    if by_package && !by_project
+    if by_package && !package
```

Comment:
Looking at the error message, this should be `if by_package && !by_project`, right?